.github/workflows/rabbitmq.yml

```diff
@@ -0,0 +1,68 @@
+name: rabbitmq
+
+on:
+    push:
+        branches-ignore: [gh-pages]
+    pull_request:
+        branches-ignore: [gh-pages]
+        paths-ignore: ['docs/**']
+
+jobs:
+
+    tests:
+
+        runs-on: ubuntu-latest
+        timeout-minutes: 30
+
+        strategy:
+            fail-fast: false
+            matrix:
+                rabbitmq: [3.5, 3.6, 3.7, 3.8]
+
+        services:
+            postgres:
+                image: postgres:10
+                env:
+                    POSTGRES_DB: test_django
+                    POSTGRES_PASSWORD: ''
+                    POSTGRES_HOST_AUTH_METHOD: trust
+                options: >-
+                    --health-cmd pg_isready
+                    --health-interval 10s
+                    --health-timeout 5s
+                    --health-retries 5
+                ports:
+                -   5432:5432
+            rabbitmq:
+                image: rabbitmq:${{ matrix.rabbitmq }}
+                ports:
+                -   5672:5672
+
+        steps:
+        -   uses: actions/checkout@v2
+
+        -   name: Set up Python 3.8
+            uses: actions/setup-python@v2
+            with:
+                python-version: 3.8
+
+        -   name: Install system dependencies
+            run: |
+                sudo rm -f /etc/apt/sources.list.d/dotnetdev.list /etc/apt/sources.list.d/microsoft-prod.list
```

Comment:
just out of interest, what are these and why do they need to be removed?

Reply:
They are also in the `ci-code.yml` workflow. They were added a few months back because that source list was causing the `apt` commands to fail. I can remove it here to see if the problem has been resolved.